packages/mui-material/src/CardHeader/CardHeader.d.ts

```diff
@@ -5,62 +5,71 @@ import { OverridableComponent, OverrideProps } from '../OverridableComponent';
 import { Theme } from '..';
 import { CardHeaderClasses } from './cardHeaderClasses';
 
+export interface CardHeaderOwnProps<
+  SubheaderTypographyComponent extends React.ElementType = 'span',
+  TitleTypographyComponent extends React.ElementType = 'span',
+> {
+  /**
+   * The action to display in the card header.
+   */
+  action?: React.ReactNode;
+  /**
+   * The Avatar element to display.
+   */
+  avatar?: React.ReactNode;
+  /**
+   * Override or extend the styles applied to the component.
+   */
+  classes?: Partial<CardHeaderClasses>;
+  /**
+   * If `true`, `subheader` and `title` won't be wrapped by a Typography component.
+   * This can be useful to render an alternative Typography variant by wrapping
+   * the `title` text, and optional `subheader` text
+   * with the Typography component.
+   * @default false
+   */
+  disableTypography?: boolean;
+  /**
+   * The content of the component.
+   */
+  subheader?: React.ReactNode;
+  /**
+   * These props will be forwarded to the subheader
+   * (as long as disableTypography is not `true`).
+   */
+  subheaderTypographyProps?: TypographyProps<
+    SubheaderTypographyComponent,
+    {
+      component?: SubheaderTypographyComponent;
+    }
+  >;
+  /**
+   * The system prop that allows defining system overrides as well as additional CSS styles.
+   */
+  sx?: SxProps<Theme>;
+  /**
+   * The content of the component.
+   */
+  title?: React.ReactNode;
+  /**
+   * These props will be forwarded to the title
+   * (as long as disableTypography is not `true`).
+   */
+  titleTypographyProps?: TypographyProps<
+    TitleTypographyComponent,
+    {
+      component?: TitleTypographyComponent;
+    }
+  >;
+}
+
 export interface CardHeaderTypeMap<
-  Props = {},
+  P = {},
```

Comment:
Do not use one-letter parameter names.
We're slowly moving away from them (especially in Base UI, but this change will come to other projects as well).

Reply:
Should I update all the other *TypeMap interfaces as well in this PR?

Reply:
We can do this in a separate PR.